src/stylesheets/core/mixins/_sidenav.scss

```diff
@@ -3,8 +3,8 @@
 
   > li {
     background-color: transparent;
-    border-top: 1px solid $color-gray;
-    font-size: $h4-font-size;
+    border-top: units(1px) solid color('base-dark');
+    font-size: scale('body', 5);
```

Comment:
I _think_ we should try to use the theme tokens as much as possible: `'sm'` in this case

Reply:
Done in 8243f73.